### ISSUE TITLE
Printing to stdout instead of returning an error or logging

### ISSUE THREAD

**Issue by wagoodman** [saferwall/pe#87]
It looks like there is some code that can output to stdout when using this library: https://github.com/saferwall/pe/blob/712c8311391d9756c8c96f05004e616e658eb5c9/dotnet.go#L730

In `anchore/syft` you can see this with:
```
syft -o json --platform windows/amd64 registry:mcr.microsoft.com/windows/servercore:ltsc2019 > /tmp/file.txt
```
```
# cat /tmp/file.txt | more
unhandled metadata table 38 File offset 0x944426 cols 5
unhandled metadata table 38 File offset 0xafdd82 cols 5
unhandled metadata table 38 File offset 0x1e720 cols 1
unhandled metadata table 38 File offset 0x3015aa cols 5
unhandled metadata table 38 File offset 0x1e718 cols 1
unhandled metadata table 38 File offset 0x30171e cols 5
unhandled metadata table 38 File offset 0x45a cols 1
unhandled metadata table 38 File offset 0x45a cols 1
unhandled metadata table 38 File offset 0x45a cols 1
unhandled metadata table 38 File offset 0x45a cols 1
unhandled metadata table 38 File offset 0x45a cols 1
unhandled metadata table 38 File offset 0x45a cols 1
unhandled metadata table 38 File offset 0x43c cols 1
unhandled metadata table 38 File offset 0x44470 cols 1
unhandled metadata table 38 File offset 0x92d34e cols 5
unhandled metadata table 38 File offset 0x583f4 cols 1
unhandled metadata table 38 File offset 0xadac8a cols 5
{
 "artifacts": [
  {
   "id": "845df080a5c1592e",
   "name": " ",
   "version": "10.0.0.0",
   "type": "dotnet",
   "foundBy": "dotnet-portable-executable-cataloger",
...
```

Ideally there shouldn't be printing to stdout in library calls, instead returning an error or writing directly to a logger object would be preferred. The reason why is because the `stdout` from syft is expected to be valid JSON, which this prevents (where a logger would write this to `stderr`).

**Comment by galamit1** [comment]
Hi, Is there any update about this issue?

**Comment by LordNoteworthy** [comment]
Hello @galamit1,

I will make a fix tonight.

**Comment by LordNoteworthy** [comment]
Fix has been release on v1.4.8, by upgrading to last version, that should not print to stdout anymore.

I changed that statement to a log msg of warning level.

This should fix it, if the problem persists, please feel free to open.

**Comment by galamit1** [comment]
Thank you!
